packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.test.ts

```diff
@@ -144,6 +148,59 @@ test("Valid request returns appointments of the right hospital", async () => {
   );
 });
 
+test("Valid request all returns appointments of the right hospital", async () => {
+  await createUser(CoordinatorRole.HOSPITAL_COORDINATOR, [
+    Hospital.ASAF_HAROFE,
+    Hospital.TEL_HASHOMER,
+  ]);
+
+  await createDonor(DONOR_ID_1, "group1");
+  await createDonor(DONOR_ID_2, "group1");
+
+  await saveAppointment(
+    PAST_BOOKED,
+    getDate(-3),
+    Hospital.ASAF_HAROFE,
+    DONOR_ID_1
+  );
+  await saveAppointment(
+    PAST_OTHER_HOSPITAL,
+    getDate(-2),
+    Hospital.ICHILOV,
+    DONOR_ID_1
+  );
+  await saveAppointment(PAST_NOT_BOOKED, getDate(-1), Hospital.TEL_HASHOMER);
+  await saveAppointment(
+    FUTURE_BOOKED,
+    getDate(3),
+    Hospital.TEL_HASHOMER,
+    DONOR_ID_2
+  );
+  await saveAppointment(
+    FUTURE_OTHER_HOSPITAL,
+    getDate(4),
+    Hospital.BEILINSON,
+    DONOR_ID_2
+  );
+  await saveAppointment(FUTURE_NOT_BOOKED, getDate(5), Hospital.TEL_HASHOMER);
+  const res = await callFunction(COORDINATOR_ID, undefined, true);
+  let appointments = res.appointments.filter((a) =>
+    ALL_APPOINTMENT_IDS.includes(a.id)
+  );
+  expect(appointments).toHaveLength(4);
+  expect(appointments[0].id).toEqual(PAST_BOOKED);
+  expect(appointments[1].id).toEqual(PAST_NOT_BOOKED);
+  expect(appointments[2].id).toEqual(FUTURE_BOOKED);
+  expect(appointments[3].id).toEqual(FUTURE_NOT_BOOKED);
```

Comment:
You can use `expect(...).toContain(...)`

Reply:
write